account-kit/react/src/components/auth/card/eoa.tsx

```diff
@@ -10,24 +10,47 @@ import type { AuthType } from "../types.js";
 import { CardContent } from "./content.js";
 import { Spinner } from "../../../icons/spinner.js";
 import { ConnectionError } from "./error/connection-error.js";
+import { EOAWallets } from "./error/types.js";
 
 interface Props {
   authStep: Extract<AuthStep, { type: "eoa_connect" }>;
 }
 
 // eslint-disable-next-line jsdoc/require-jsdoc
 export const EoaConnectCard = ({ authStep }: Props) => {
+  const { setAuthStep } = useAuthContext();
+
+  if (authStep.error) {
+    return (
+      <ConnectionError
```

Comment:
Does this need to live inside of the CardContent below?

Reply:
Yeah we went this direction to prevent prop drilling all the various error props and actions to be taken on error. Keeping it a level higher helps us decouple the error from the default states.